src/org/skypro/skyshop/product/BestResultNotFound.java

```diff
@@ -0,0 +1,23 @@
+package org.skypro.skyshop.product;
+
+public class BestResultNotFound extends Exception {
```

Comment:
классы-исключения лучше называть с Exception на конце
```suggestion
public class BestResultNotFoundException extends Exception {
```